sovtokenfees/sovtokenfees/test/3pc_batching/test_reverting_batches.py

```diff
@@ -0,0 +1,65 @@
+import json
+
+from plenum.test.delayers import cDelay
+
+from plenum.test.stasher import delay_rules
+from sovtoken import TOKEN_LEDGER_ID
+from sovtokenfees.test.helper import add_fees_request_with_address, get_head_hash_for_pool, \
+    get_uncommitted_txns_count_for_pool
+
+from plenum.test.helper import sdk_send_signed_requests, sdk_get_and_check_replies, sdk_send_random_and_check
+
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data
+
+from plenum.test.pool_transactions.helper import sdk_add_new_nym
+
+from plenum.test import waits
+
+
+def test_revert_batches_with_fees_unset_fees_without_fee(looper, helpers,
```

Comment:
It would be great to have a test when we
- delay COMMITs on all nodes
- apply a txn with FEES 
- apply a txn without FEES 
- apply a txn with FEES 
- reset delays
- make sure everything is ordered and the pool is equal and functional

Reply:
Also it would be great to have the following same test
* delay COMMITs on all Nodes
* apply a txn with FEES 
* apply a txn without FEES
* apply a txn with FEES
* start catchup
* make sure the pool is equal and functional